plugin/src/test/java/io/jenkins/plugins/coverage/metrics/steps/CoverageBuildActionTest.java

```diff
@@ -88,4 +114,32 @@ void shouldCreateViewModel() {
         assertThat(action.getTarget()).extracting(CoverageViewModel::getNode).isSameAs(root);
         assertThat(action.getTarget()).extracting(CoverageViewModel::getOwner).isSameAs(action.getOwner());
     }
+
+    @Test
+    void shouldReturnPositiveTrendForLineMetric() {
+        CoverageBuildAction action = createCoverageBuildActionWithDelta(Metric.LINE);
+        int trend = action.getTrend(Baseline.PROJECT, Metric.LINE);
+        assertThat(trend).isEqualTo(1);
```

Comment:
Can you make the delta values visible here? Otherwise it is hard to understand the test. It would also make sense to have in one test the boundaries used that return 0, >0, and <0